README_ja-JP.md

```diff
@@ -75,23 +75,27 @@ MAAは、MAA Assistant Arknightsです。
 
 ## よくある質問
 
-- 起動するとき、ソフトウェアがクラッシュする
-- 接続エラー、adbパスがわからない場合
-- 認識エラー、またはオペレーション開始したあとで反応がない
-- カスタムポート設定について
+- 起動する際にソフトウェアがクラッシュする。
+- 接続エラー、adbパスがわからない場合。
+- 認識エラー、またはオペレーション開始した後に反応がない。
+- カスタムポート設定について。
 
 [よくある質問](docs/ja-jp/1.2-よくある質問.md)を参照してください。
 
 ## サーバーに応じてサポートされる機能
 
 - アメリカサーバー 
-  作戦、戦友訪問、レワード収集、統合戦略（ベータ版）、公開求人認識の機能がサポートされている。[README](resource/global/YoStarEN/readme.md)を参照してください。
+  作戦、戦友訪問、レワード収集、統合戦略（ベータ版）、公開求人認識の機能がサポートされている。
+  <br>サポート内容については[README](resource/global/YoStarEN/readme.md)を参照してください。
 - 日本サーバー 
-  作戦、自動公開求人、戦友訪問、レワード収集、統合戦略（ベータ版）、公開求人認識の機能がサポートされている。[README](resource/global/YoStarJP/readme.md)を参照してください。
+  作戦、自動公開求人、戦友訪問、レワード収集、統合戦略（ベータ版）、公開求人認識の機能がサポートされている。
+  <br>サポート内容については[README](resource/global/YoStarJP/readme.md)を参照してください。
 - 韓国サーバー 
-  基本的な作戦機能がサポートされている。[README](resource/global/YoStarKR/readme.md)を参照してください。
+  基本的な作戦機能のみサポートされている。
+  <br>サポート内容については[README](resource/global/YoStarKR/readme.md)を参照してください。
 - TXWYサーバー  
-  基本的な作戦、自動公開求人、公開求人認識の機能がサポートされている。[README](resource/global/txwy/readme.md)を参照してください。
+  基本的な作戦、自動公開求人、公開求人認識の機能がサポートされている。
+  <br>サポート内容については[README](resource/global/txwy/readme.md)を参照してください。
```

Comment:
In markdown format, you can add two spaces at the end of a line instead of a `<br>` to indicate a line break.